### PR TITLE
Configurability and batch running updates

### PR DESCRIPTION
**Pull request by matthewkenzie**

### CODE REVIEW

File: core/include/UtilsConfig.h [hunk 1]
```diff
@@ -106,6 +106,9 @@ namespace Utils
 		hfagLP2011,
 		hfagCHARM2015,
     hfagCKM2016,
+		hflav,
```
Comment (tmombaecher) [inline core/include/UtilsConfig.h]: I don't really see why the names of all combinations used should be hardcoded here. But ok

**Comment by tmombaecher** [comment]
but that's all. Great progress! thanks!

File: core/include/UtilsConfig.h (hunk 1 above)
Comment (matthewkenzie) [reply core/include/UtilsConfig.h]: Yes, frankly it's totally dumb, but sort of there for historical reasons. It really should be replaced with just simple strings as it is elsewhere. I shouldn't keep using this methodology but I do so to keep consistent with older combinations. Essentially in the world average combinations there are some case statements which I think would take quite a bit of rewriting. However you have incentivised me to abandon this approach.

**Comment by matthewkenzie** [comment]
> I found it more elegant to use `xrdcp -p` instead of `mkdir ...` as it does not rely on any kind of mount. Only needs an authentication token (kerberos), which on lxplus is a given

This mkdir command is actually to make the local directory that you will eventually synch things back to (i.e. I use softlinks to Eos for this) but it needs to be made because when you are not writing the output to eos (with --batcheos option) then you want the output file copied back locally and so you need this directory in place.

Indeed the xrdcp will take care of making the directory on Eos (but that is not this one).

**Comment by tmombaecher** [comment]
> > I found it more elegant to use `xrdcp -p` instead of `mkdir ...` as it does not rely on any kind of mount. Only needs an authentication token (kerberos), which on lxplus is a given
> 
> This mkdir command is actually to make the local directory that you will eventually synch things back to (i.e. I use softlinks to Eos for this) but it needs to be made because when you are not writing the output to eos (with --batcheos option) then you want the output file copied back locally and so you need this directory in place.
> 
> Indeed the xrdcp will take care of making the directory on Eos (but that is not this one).

Ok, but why removing the `-p` in the xrdcp command? If I read this correctly, the `--batcheos` option will not work properly, as the directory on eos won't exist prior to the copy command.

**Comment by matthewkenzie** [comment]
Ah yes. You may well be write. I didn't spot that. That line will need to go back in.

**Comment by tmombaecher** [comment]
> > > I found it more elegant to use `xrdcp -p` instead of `mkdir ...` as it does not rely on any kind of mount. Only needs an authentication token (kerberos), which on lxplus is a given
> > 
> > 
> > This mkdir command is actually to make the local directory that you will eventually synch things back to (i.e. I use softlinks to Eos for this) but it needs to be made because when you are not writing the output to eos (with --batcheos option) then you want the output file copied back locally and so you need this directory in place.
> > Indeed the xrdcp will take care of making the directory on Eos (but that is not this one).
> 
> Ok, but why removing the `-p` in the xrdcp command? If I read this correctly, the `--batcheos` option will not work properly, as the directory on eos won't exist prior to the copy command.

Sorry, didn't read properly. please dismiss the comment

**Pull request merged**